tests/test_2qbits_gates.py

```diff
@@ -109,18 +108,16 @@ def check_cz_with_heralds_or_ancillaries(processor, herald_states, error=1E-6):
                 assert pytest.approx(modulus) == 0
     return modulus_value
 
-
```

Comment:
With PEP8, there must be two lines between methods outside of Class, can you add back a line please ?